docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx

```diff
@@ -1,34 +1,47 @@
 ---
 title: Instrument Caches
 sidebar_order: 1000
-description: "Learn how to manually instrument your code to use Sentry's Cache module. "
+description: "Learn how to manually instrument your code to use Sentry's Cache module."
+supported:
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.bun
+  - javascript.deno
+  - javascript.nextjs
+  - javascript.astro
+  - javascript.sveltekit
+  - javascript.remix
 ---
 
 A cache can be used to speed up data retrieval, thereby improving application performance. Because instead of getting data from a potentially slow data layer, your application will be getting data from memory (in a best case scenario). Caching can speed up read-heavy workloads for applications like Q&A portals, gaming, media sharing, and social networking.
 
-Sentry offers a [cache-monitring dashboard](https://sentry.io/orgredirect/organizations/:orgslug/performance/caches/) that can be auto-instrumented using Sentry's redis integration (more coming soon).
-
+Sentry offers a [cache-monitoring dashboard](https://sentry.io/orgredirect/organizations/:orgslug/performance/caches/) that can be auto-instrumented using Sentry's redis integration (more coming soon).
 
 ## Manual Instrumentation
 
-If you're using anything other than Sentry's redis integration, you'll need to manually instrument the [Cache Module](<(https://sentry.io/orgredirect/organizations/:orgslug/performance/caches/)>) by following the steps below.
+If you're using anything other than Sentry's Redis integration, you'll need to manually instrument the [Cache Module](<(https://sentry.io/orgredirect/organizations/:orgslug/performance/caches/)>) by following the steps below.
 
 You'll need to create two spans - one indicating that something is being put into the cache, and a second one indicating that something is being fetched from the cache.
 
-Make sure that there's a transaction running when you create the spans. If you're using a web framework those transactions will be created for you automatically. See <PlatformLink to="/performance/">Performance Monitoring</PlatformLink> for more information.
-
-For detailed information about which data can be set, see the [Cache Module Developer Specification](https://develop.sentry.dev/sdk/performance/modules/caches/).
-
+<PlatformSection supported={["javascript.node", "javascript.deno", "javascript.bun"]}>
```

Comment:
```suggestion
<PlatformCategorySection supported={["server"]}>
```

I think this is better, or do we not want to show this for fastify etc?

Reply:
hmm or I guess, this is fine, should only be shown for framework-less SDK?

Reply:
yeah that was my thinking. Doesn't make sense in guides where we already have auto-instrumented request spans